lib/backpex/item_actions/item_action.ex

```diff
@@ -22,10 +22,10 @@ defmodule Backpex.ItemAction do
   @callback fields() :: list()
 
   @doc """
-  Initial change. The result will be passed to `Backpex.ItemAction.changeset/3` in order to generate a changeset.
+  Initial change. The result will be passed to [`Backpex.ItemAction.changeset/3`](Backpex.ItemAction.html#c:changeset/3) in order to generate a changeset.
```

Comment:
This should work [according to the docs](https://hexdocs.pm/ex_doc/readme.html#auto-linking):

```suggestion
  Initial change. The result will be passed to `c:Backpex.ItemAction.changeset/3` in order to generate a changeset.
```

Or maybe even this because it is in the same module?

```suggestion
  Initial change. The result will be passed to `c:changeset/3` in order to generate a changeset.
```

Reply:
Works 👍